blackberggroup/VA-iEX
# feat: update hero background styles

## What type of PR is this? (check all applicable)


## Description

- Remove hero pattern bg
- Add circle gradient to hero
- update <body> with gradient background

## Related Tickets & Documents
[- Monday Task #](https://blackberg-group.monday.com/boards/1931719392/pulses/7207568083)

## QA Instructions, Screenshots, Recordings

- View gradient background 
- Verify bg pattern removed from hero

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/iex-2024/hero.html
 - _site/assets/css/styles.css
 - _site/assets/js/animations/hero-animations.js
 - _site/assets/sass/_uswds-theme-custom-styles.scss
 - _site/assets/sass/vha/pages/iex-2023/hero/_index.scss
 - _site/index.html
 - assets/css/styles.css
 - assets/js/animations/hero-animations.js
 - assets/sass/_uswds-theme-custom-styles.scss
 - assets/sass/vha/pages/iex-2023/hero/_index.scss
